driver/src/main/java/org/kaazing/k3po/driver/internal/netty/bootstrap/http/HttpServerChannelSink.java

```diff
@@ -51,7 +51,7 @@
 public class HttpServerChannelSink extends AbstractServerChannelSink<HttpServerChannel> {
 
     private final ConcurrentNavigableMap<URI, HttpServerChannel> httpBindings;
```

Comment:
I think httpBindings should be changed to Map<ChannelAddress, HttpServerChannel>. Otherwise, if there are two accepts 1: URI 2: URI+transport accept option , they run into conflicts